fastlane/swift/LaneFileProtocol.swift

```diff
@@ -19,15 +19,13 @@ public protocol LaneFileProtocol: class {
     static func runLane(named: String, parameters: [String : String]) -> Bool
 
     func recordLaneDescriptions()
-    func beforeAll(currentLane: String, parameters: [String : String])
     func beforeAll()
     func afterAll(currentLane: String)
     func onError(currentLane: String, errorInfo: String)
 }
 
 public extension LaneFileProtocol {
     var fastlaneVersion: String { return "" } // default "" because that means any is fine
-    func beforeAll(currentLane: String, parameters: [String : String]) { } // no op by default
```

Comment:
Can we keep this one in here? This was added in a recent PR. Its the same as. It looks like its duplicated but `beforeAll` was implemented first and some people wanted the lane and parameters to get sent over like it does in the Ruby version so this was added 😇

Reply:
Maybe this actually requires a rebase from master?

Reply:
Probably, because the base of this PR was the other PR... 1.137.0